apps/judicial-system/web/src/routes/PublicProsecutor/Indictments/Overview/Overview.tsx

```diff
@@ -39,15 +39,10 @@ import { strings } from './Overview.strings'
 type VisibleModal = 'REVIEWER_ASSIGNED' | 'DEFENDANT_VIEWS_VERDICT'
 
 export const isDefendantInfoActionButtonDisabled = (defendant: Defendant) => {
-  switch (defendant.serviceRequirement) {
-    case ServiceRequirement.NOT_APPLICABLE:
-    case ServiceRequirement.NOT_REQUIRED:
-      return true
-    case ServiceRequirement.REQUIRED:
-      return defendant.verdictViewDate !== null
-    default:
-      return false
-  }
+  return (
+    defendant.serviceRequirement === ServiceRequirement.NOT_REQUIRED ||
+    Boolean(defendant.verdictViewDate)
+  )
```

Comment:
This modification ensures that even if setting the verdict view date for fines (viðurlagaákvörðun) fails, the user can set it manually here. The date may be wrong, giving the defendant more time to appeal than requred, but that will. not have any serious consequences.